Sources/Controllers/2.0/OAMapSettingsCategoryScreen.mm

```diff
@@ -189,17 +196,26 @@ - (void) tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
 
 - (void) mapSettingSwitchChanged:(id)sender
 {
-     UISwitch *switchView = (UISwitch*)sender;
-     if (switchView) {
-         OAMapStyleParameter *p = parameters[switchView.tag];
-         if (p) {
-             dispatch_async(dispatch_get_main_queue(), ^{
-                 p.value = switchView.isOn ? @"true" : @"false";
-                 [styleSettings save:p];
-             });
-         }
-     }
+    UISwitch *switchView = (UISwitch*)sender;
+    if (!switchView)
```

Comment:
It is not needed to change code style to "if (!) return" here. Left it "if () { if () {..."